torch/functional.py

```diff
@@ -884,7 +884,7 @@ def tensordot(a, b, dims=2):
     Args:
       a (Tensor): Left tensor to contract
       b (Tensor): Right tensor to contract
-      dims (int or tuple of two lists of integers): number of dimensions to
+      dims (int or tuple of two lists of int): number of dimensions to
```

Comment:
nit: I think `int or Tuple[List[Int]]` makes it a clearer (all of the "of"s throw me off a little)

Reply:
Fixed, but I used `Tuple[List[int]] of two lists` to indicate that the tuple must be a pair.

Reply:
Sorry, I didn't read that part. `Tuple[List[int]] of two lists` sounds good to me (y)